src/Documentation/Getting-Started-With-Orleans/Grain-Persistence.md

```diff
@@ -134,6 +134,9 @@ A quick way to test this is to (see in the aforementioned the few commented line
 The ADO.NET persistence has functionality to version data and define arbitrary (de)serializers with arbitrary application rules and streaming, but currently
 there is no method to expose them to application code. More information in [ADO.NET Persistence Rationale](#ADONETPersistenceRationale).
 
+__NB for Microsoft SQL Server users:__
+For the ADO.NET Storage Provider to work with SQL Server, you need to add [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient/) NuGet package to your project.
```

Comment:
Yes. This is the case for all providers. It used to be that for SQL Server the provider was included by default in the framework, whereas for others one had to get it from somewhere already (e.g. from Nuget). The problem might be that the SQL Server users haven't accustomed to consider this thing, so they should perhaps be explicitly mentioend. Though it doesn't hurt to mention that in general too. So maybe mention the client needs to acquired by some means, Nuget is a good source, and in .NET Core that applies also to SQL Server.